roles/pulp_distribution/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+pulp_url: http://localhost:24817
```

Comment:
24816/24817 are only bound to 127.0.0.1 - maybe we should default to using 80 (Nginx forwarder) for most cases?

Reply:
Well localhost is 127.0.0.1, no?

Anyway, a typical Pulp in one with docker networking gives you port 8080.